handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -246,13 +246,24 @@
                             SSL.setKeyMaterial(ssl, cert, key);
                             supportsKeyManagerFactory = true;
                             try {
-                                useKeyManagerFactory = AccessController.doPrivileged(new PrivilegedAction<Boolean>() {
-                                    @Override
-                                    public Boolean run() {
-                                        return SystemPropertyUtil.getBoolean(
-                                                "io.netty.handler.ssl.openssl.useKeyManagerFactory", true);
+                                boolean propertySet = SystemPropertyUtil.contains(
+                                        "io.netty.handler.ssl.openssl.useKeyManagerFactory");
+                                if (!IS_BORINGSSL) {
+                                    useKeyManagerFactory = SystemPropertyUtil.getBoolean(
+                                            "io.netty.handler.ssl.openssl.useKeyManagerFactory", true);
+
+                                    if (propertySet) {
+                                        logger.debug("System property " +
```

Comment:
debug log? Nobody will see that...

Reply:
Everything else seems too much

Reply:
This seems appropriate for the warning level. "This thing you're using will not do anything in the future." I'd want to know about that.

But if we keep it at debug level... what's the point? Nobody will see it, so why bother?

Reply:
@ejona86 so you say we should either switch to `WARNING` or remove the logging at all ?

Reply:
Basically, yeah. INFO/WARNING would be fine. Maybe we only issue the warning for tcnative today, since that's when the option is ignored.

Reply:
let me just use INFO then...